certbot/compat/os.py

```diff
@@ -0,0 +1,20 @@
+"""
+This compat modules extends native capabilities of core os module to handle correctly platform
+specific operations (eg. chown, chmod, geuid).
+This module is intended to replace standard os module throughout certbot projects (except acme).
+"""
+from __future__ import absolute_import
+
+# Expose everything from standard os package to make current package a complete replacement of os.
+from os import *  # type: ignore  # pylint: disable=wildcard-import,unused-wildcard-import,redefined-builtin,os-module-forbidden
+
+# Monkey patch ourselves to get os attributes that are not in __all__ (so not from os import *).
+import os as std_os  # pylint: disable=os-module-forbidden
+import sys as std_sys
+ourselves = std_sys.modules[__name__]
+for attribute in dir(std_os):
+    if not hasattr(ourselves, attribute):
```

Comment:
If we get rid of the `from os import *` at the top, I was initially unsure if we needed this line, but I think we do. I don't think we want to overwrite attributes like `__doc__`, `__file__`, and `__name__`. Since I was initially unsure, can you add a comment explaining why we need this?

Reply:
Yes, I will write about this, additionally to the comment needed as explained in the above discussion.

Reply:
Updated.